src/__experimental__/components/number/number.stories.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+import { StateDecorator, Store, State } from '@sambego/storybook-state';
+import {
+  boolean,
+  number
+} from '@storybook/addon-knobs';
+import Number from './number.component';
+import notes from './notes.md';
+import { StoryHeader, StoryCode } from '../../../../.storybook/style/storybook-info.styles';
+import getTextboxStoryProps from '../textbox/textbox.stories';
+
+const store = new Store(
+  {
+    value: ''
+  }
+);
+
+const setValue = (ev) => {
+  action('onChange')(ev);
+  store.set({ value: ev.target.value });
+};
+
+storiesOf('Experimental/Number Input', module)
+  .addDecorator(StateDecorator(store))
+  .add('default', () => {
+    const onChangeDeferred = boolean('onChangeDeferred', false);
+    const deferTimeout = onChangeDeferred ? number('deferTimeout') : undefined;
+
+    return (
+      <Number
+        { ...getTextboxStoryProps() }
+        value={ store.get('value') }
+        onChange={ setValue }
+        onKeyDown={ action('onKeyDown') }
+        onChangeDeferred={ onChangeDeferred ? action('onChangeDeferred') : undefined }
+        deferTimeout={ deferTimeout }
+      />
+    );
+  }, {
+    info: {
+      text: (
```

Comment:
We're tending to move all docs to a `documentation` directory in the component, and then importing them.

Reply:
could you add that to our code convention?

https://github.com/Sage/carbon/wiki/Storybook